defaults/aws_config.yaml

```diff
@@ -9,12 +9,12 @@ regions_data:
     security_group_ids: 'sg-c5e1f7a0'
     subnet_id: 'subnet-ec4a72c4'
     ami_id_loader: 'ami-050c06c3e20036edc' # Loader dedicated AMI
-    ami_id_monitor: 'ami-01d6b0d7e0d98969b' # Monitor dedicated AMI
+    ami_id_monitor: 'ami-0d52990d49810f04e' # clean centos
   eu-west-1:
     security_group_ids: 'sg-059a7f66a947d4b5c'
     subnet_id: 'subnet-088fddaf520e4c7a8'
     ami_id_loader: 'ami-0006222380fb72d8d' # Loader dedicated AMI
-    ami_id_monitor: 'ami-0f478f18d613e3662' # Monitor dedicated AMI
+    ami_id_monitor: 'ami-092d25a6b3407805d' # clean centos
```

Comment:
Amos, where did you get those AMIs from?
they are not in official CentOS page: https://wiki.centos.org/Cloud/AWS

Reply:
command to get CentOS AMIs 
`aws --region us-east-1 ec2 describe-images --owners aws-marketplace --filters Name=product-code,Values=aw0evgkw8e5c1q413zgy5pjce`